time_to_first_response.py

```diff
@@ -109,6 +102,25 @@ def measure_time_to_first_response(
     return None
 
 
+def ignore_comment(
+    issue_user: github3.users.User,
+    comment_user: github3.users.User,
+    ignore_users: List[str],
+    comment_created_at: datetime,
+    ready_for_review_at: Union[datetime, None],
+) -> bool:
+    """Check if a comment should be ignored."""
+    return (
+        # ignore comments by IGNORE_USERS
+        comment_user.login in ignore_users
+        # ignore comments by bots
+        or comment_user.type == "Bot"
```

Comment:
Were you able to test this one out? I was worried by the documentations description:
> A string representing the type of User account this. **In all cases should be “User”**

Reply:
https://github3.readthedocs.io/en/latest/api-reference/users.html#github3.users.ShortUser.type

Reply:
I have not read the documentation, but I have confirmed that a "bot" is returned when the API is called.

For example.

```sh
% gh api \
  -H "Accept: application/vnd.github+json" \
  -H "X-GitHub-Api-Version: 2022-11-28" \
  /users/codesee-maps\[bot\]
{
  "login": "codesee-maps[bot]",
  ...
  "type": "Bot",
  ...
}
```

As a side note, I hope the api documentation clearly states this. However, the Examples only mentions "User".

https://docs.github.com/en/rest/users/users?apiVersion=2022-11-28#get-a-user